src/style.css

```diff
@@ -1 +1,44 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: start;
+}
+
+.stars__star {
+  display: flex;
+  justify-content: center;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  flex-shrink: 0;
+  background-position: center;
+  margin-right: 4px;
```

Comment:
You are using margins on both sides (right and left) of the element. To avoid potential margin collapse, it's better to be consistent with your margins. Consider using padding or restructure your layout.